process-document/src/main/java/com/ritense/processdocument/service/impl/CamundaProcessJsonSchemaDocumentDeploymentService.java

```diff
@@ -114,8 +114,8 @@ private void createProcessDocumentLink(String documentDefinitionName, ProcessDoc
         }
 
         contextService.findAll(Pageable.unpaged()).forEach(context -> {
-            context.getProcesses().removeIf(contextProcess -> contextProcess.getProcessDefinitionKey().equals(item.getProcessDefinitionKey()));
             if (Boolean.TRUE.equals(item.getProcessIsVisibleInMenu())) {
+                context.getProcesses().removeIf(contextProcess -> contextProcess.getProcessDefinitionKey().equals(item.getProcessDefinitionKey()));
```

Comment:
This is not going to work when someone tries to disable the `processIsVisibleInMenu` through the auto-deployment file. 
Because when changed to `"processIsVisibleInMenu": false`, the context isn't saved here.

Reply:
But it is not working now either. As I do not get my tasks to show up on the dashboard, even if I have "processIsVisibleInMenu": false because it always removes it. The context menu also has 3 options and not two, so the code is incorrect either way. 

What should the processIsVisibleInMenu actually do?

Reply:
Just looked into this. The context menu shows 3 options, but in reality only has 2 options. Which is visible when opening the network tab when saving the contexts.

The `processIsVisibleInMenu = true` is the same as the option `process and tasks`.

I've made a fix that won't in this PR: https://github.com/valtimo-platform/valtimo-backend-libraries/pull/610/files

Reply:
@Klaas-Ritense @nielserkamp-ritense The PR Klaas mentioned has been merged. Does that solve the issue and make this PR obsolete?